src/client/mod.rs

```diff
@@ -143,6 +143,58 @@ impl<T: Transport, L: LoginCredentials> TwitchIRCClient<T, L> {
             .await
     }
 
+    /// Ban a user with a optional reason in the given Twitch channel.
+    pub async fn ban(
+        &self,
+        channel_login: String,
+        user: String,
+        reason: Option<String>,
+    ) -> Result<(), Error<T, L>> {
+        self.privmsg(
+            channel_login,
+            format!("/ban {} {}", user, reason.unwrap_or_default()),
+        )
+        .await
+    }
+
+    /// Unban a user in the given Twitch channel.
+    pub async fn unban(&self, channel_login: String, user: String) -> Result<(), Error<T, L>> {
+        self.privmsg(channel_login, format!("/unban {}", user))
+            .await
+    }
+
+    /// Timeout a user with a optional reason in the give Twitch channel.
+    ///
+    /// If no duration is specified the Twitch default of ten minutes is used.
+    ///
+    /// Duration is not checked for validity. It must be a positive integer, a
+    /// optional time unit after the number must be one of `s`, `m`, `h`, `d`,
+    /// `w` where `s` is the default. The maximum is two weeks.
```

Comment:
I would prefer if the duration was specified as a `Duration` instance, instead of a string. You can just `/timeout user <number> reason` where `<number>` is a number of seconds.

Reply:
since `chrono` is already a dependency, should I use `chrono::Duration` or `std::time::Duration`?

Reply:
I would prefer `std::time::Duration` since it seems more universal to me to use the stdlib rather than a third-party library